mypy/checker.py

```diff
@@ -6929,7 +6927,7 @@ def reduce_conditional_maps(type_maps: list[tuple[TypeMap, TypeMap]]) -> tuple[T
     We only retain the shared expression in the 'else' case because we don't actually know
     whether x was refined or y was refined -- only just that one of the two was refined.
     """
-    if len(type_maps) == 0:
+    if not type_maps:
```

Comment:
This one I'd also prefer to keep as-is, as it retains symmetry with the `elif` condition immediately below, which has to use `len`